lambda_functions/analyzer/README.rst

```diff
@@ -4,38 +4,61 @@ This Lambda function is the core of BinaryAlert. Each invocation downloads one o
 S3, scans them against all available YARA rules, and forwards any matches to Dynamo and SNS.
 
 
-Updating YARA-Python
---------------------
-The `yara-python <https://github.com/VirusTotal/yara-python>`_ library is natively compiled.
-It must therefore be pre-built on an Amazon Linux AMI in order to run in Lambda.
-This has already been done for you: ``yara_python_3.6.3.zip`` contains the
-pre-built ``yara_python`` library (v3.6.3) for the Lambda environment and is automatically bundled
-on every deploy.
+Updating YARA Binaries
+----------------------
+The YARA libaries used by BinaryAlert are natively compiled, and must therefore be pre-built on an
+Amazon Linux AMI in order to run in Lambda. This has already been done for you:
+``yara3.7.0_yextend1.5.zip`` contains the pre-built
+`yara_python <https://github.com/VirusTotal/yara-python>`_ (v3.7.0) and
+`yextend <https://github.com/BayshoreNetworks/yextend>`_ (v1.5) libraries for the Lambda environment
+and is automatically bundled with every deploy.
 
 If, however, you need to update or re-create the zipfile, SSH to an EC2 instance running the
 `AWS Lambda AMI <http://docs.aws.amazon.com/lambda/latest/dg/current-supported-versions.html>`_
```

Comment:
have you considered automating this process with Packer or Chef?

Reply:
@strcrzy and @ryandeivert have looked at this. Packer would be great, but we don't update the dependencies often enough to justify spending the time to automate it (yet)

Definitely something to consider! I might open an issue for it

Reply:
that's fair, I only suggest because it's really easy to do